resources/views/welcome.blade.php

```diff
@@ -62,50 +57,33 @@
                 margin-bottom: 30px;
             }
         </style>
-    </head>
-    <body>
-        <div class="flex-center position-ref full-height">
-            @if (Route::has('login'))
-                <div class="top-right links">
-                    @auth
-                        <a href="{{ url('/home') }}">Home</a>
-                    @else
-                        <a href="{{ route('login') }}">@lang('general.login')</a>
-
-                        @if (Route::has('register'))
-                            <a href="{{ route('register') }}">@lang('general.register') @lang('general.register_collegist')</a>
-                            <a href="{{ route('register.guest') }}">@lang('general.register') @lang('general.register_guest')</a>
-                        @endif
-                    @endauth
-                    
-                     @if (App::isLocale('en'))
-                        <a class="nav-link" href="{{ route('setlocale', 'hu') }}">{{ __('HU') }}</a>
-                    @else
-                        <a class="nav-link" href="{{ route('setlocale', 'en') }}">{{ __('EN') }}</a>
-                    @endif
-                </div>
-            @endif
+@endsection
 
-            <div class="content">
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+        <div class="content">
                 <div class="title m-b-md">
                     Urán 2.0
                 </div>
 
                 <div class="links">
                     <a href="">
-                    @lang('main.better')</a>
+                    @lang('main.better')</a><br/>
```

Comment:
These changes look bad on PC. Maybe handling mobile and non-mobile differently would be a better approach?